lib/value-pairs/value-pairs.c

```diff
@@ -896,6 +896,16 @@ value_pairs_new(GlobalConfig *cfg)
   vp->patterns = g_ptr_array_new();
   vp->transforms = g_ptr_array_new();
 
+  if (cfg_is_config_version_older(cfg, VERSION_VALUE_4_0))
```

Comment:
We might do that just to show the intent here, but my understanding is that we don't have to. VERSION_VALUE_4_0 would be mapped to 0x0800 in the case of PE.

But you may be right this would be just make things more difficult to understand.

Reply:
Yeah, it works both ways. I was just wondering, but I'm fine with `VERSION_VALUE_4_0` as well. :)